sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -1248,7 +1280,7 @@ private object Get extends Val.Builtin("get", Array("o", "f", "default", "inc_hi
       str.isEmpty
     },
     builtin("trim", "str") { (_, _, str: String) =>
-      str.trim
+      str.replaceAll("^[ \t\n\f\r\u0085\u00A0']+", "").replaceAll("[ \t\n\f\r\u0085\u00A0']+$", "")
```

Comment:
Because the regex is a constant here, I think we can factor these out into a static Pattern instance in a companion and then use it's `replaceAll`: that will save a ton of allocations.